lib/rspec/mocks/verifying_proxy.rb

```diff
@@ -36,6 +36,7 @@ def add_message_expectation(method_name, opts={}, &block)
       end
 
       def ensure_implemented(method_name)
+        return if Mocks.configuration.temporarily_suppressing_partial_double_verification
```

Comment:
This is defined in the `VerifyingProxyMethods` module which, if I'm not mistaken, applies to normal verifying doubles (e.g. `instance_double` and `class_double`) in addition to partial doubles.  As such, I think there's a bug here: while the config option is meant to only affect partial doubles, it looks like your implementation would suppress it for normal doubles, too.

I think the solution is to move these checks into methods defined in `VerifyingPartialDoubleProxy` -- although you might have to define new ones where the checks can go (e.g. with a call to `super()`)